TODO.md

```diff
@@ -5,8 +5,8 @@ Pass the name of the module to the `ModuleName` parameter. The script will:
 
 * Rename this file to TODO.md (there are additional steps to take afterward)
 * Puts the default README.md file in place.
-* Rename every file that has `MODULE_NAME` in its name, replacing `MODULE_NAME` with the module name.
-* Replaces `MODULE_NAME` in every file with the module name.
+* Rename every file that has `Carbon.Windows.HttpServer` in its name, replacing `Carbon.Windows.HttpServer` with the module name.
```

Comment:
Remove this file?

Reply:
I cleared out all the completed contents. I'm finding a like a TODO.md file to note what to do when updating to the next minor and next major releases.